.eslintrc.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
   rules: {
     'no-shadow': 'off',
     '@typescript-eslint/no-shadow': 'error',
+    '@typescript-eslint/indent': 'off',
```

Comment:
I assume this is disabled because it's recommended to disable it when using `prettier`? The same applies to a lot of rules that remain disabled it seems :thinking: At least according to https://github.com/prettier/eslint-config-prettier/blob/master/index.js anyway.

Reply:
This was disabled because it was the most tangible issue that conflicted when running both eslint and Prettier. We should add all of those rules in a separate PR.